File: yeongseong/3week/아이스 아메리카노.js
function solution(money) {
    // 커피 한잔 가격
    const coffee = 5500;
    // 구매 가능한 커피의 개수
    const number = Math.floor(money / coffee);
    // 거스름돈
    const change = money % coffee;
    
    return [number, change];
}

File: dongmin/week3/아이스 아메리카노.java
class Solution {
    public int[] solution(int money) {
        int[] answer = new int[2];
        int i =0;
        while(money>=5500)
        {
            money-=5500;
            i++;
        }
        answer[0] = i;      
        answer[1] = money;
        return answer;
    }
}